built-in-references/KubernetesService/container-no-privilege/limited-preview/azurepolicy.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "displayName": "[Limited Preview]: Do not allow privileged containers in AKS",
+    "policyType": "BuiltIn",
+    "mode": "Microsoft.ContainerService.Data",
+    "description": "This policy does not allow privileged containers creation in an Azure Kubernetes Service cluster. Limited Preview policies only work for registered subscriptions. To register, please go to https://aka.ms/akspolicyonboarding. For instruction on using this policy, please go to https://aka.ms/akspolicydoc.",
```

Comment:
container (singular)

Reply:
A deployment spec can have more than one container and hence using containers is fine. Also the text is reviewed by David.